guides/testing.md

```diff
@@ -20,60 +44,92 @@ Concrete test cases are defined in `it` blocks. An optional (but strongly recomm
+
+Each example can contain multiple expectations, but it should test only one specific behaviour.
+
+## Expectations
 
 Expectations define if the value being tested (*actual*) matches a certain value or specific criteria.
 
-| Matcher        | Description |
-| --             | -- |
-| `eq(value)`    | passes if actual equals *value* (`==`) |
-| `be(value)`    | passes if actual and *value* are identical (`.same?`) |
-| `be_nil`       | passes if actual is nil (`== nil`) |
-| `be_true`      | passes if actual is true (`== true`) |
-| `be_false`     | passes if actual is false (`== false`) |
-| `be_truthy`    | passes if actual is truthy (neither `nil` nor `false`) |
-| `be_falsey`    | passes if actual is falsy (`nil` or `false`) |
-| `be < value`  | passes if actual is lesser than *value* (`<`) |
-| `be <= value` | passes if actual is lesser than or equal *value* (`<=`) |
-| `be > value`  | passes if actual is greater than *value* (`>`) |
-| `be >= value` | passes if actual is greater than or equal *value* (`>=`) |
-| `be_close(expected, delta)` | passes if actual is within *delta* of *expected* |
-| `contain`      | passes if actual includes *expected* (`.includes?`) |
-| `match(value)` | passes if actual matches *value* (`=~`) |
-| `be_a(type)`   | passes if actual is of type *type* (`is_a?`) |
-| `expect_raises(&block)` | runs the block and passes if it raises an exception |
-| `expect_raises(klass, &block)` | runs the block and passes if it raises an exception of type *klass* |
-| `expect_raises(klass, message, &block)` | runs the block and passes if it raises an exception of type *klass* and the error message contains *message* |
+### Equivalence, Identity and Type
+```crystal
+actual.should eq(expected)    # passes if actual == expected
+actual.should be(expected)    # passes if actual.same?(expected)
+actual.should be_a(expected)  # passes if actual.is_a?(expected)
+actual.should be_nil          # passes if actual.nil?
+```
+
+### Truthiness
+```crystal
+actual.should be_true         # passes if actual == true
+actual.should be_false        # passes if actual == false
+actual.should be_truthy       # passes if actual is truthy (neither nil nor false)
+actual.should be_falsey       # passes if actual is falsey (nil or false)
+```
+
+### Comparisons
+```crystal
+actual.should be <  expected  # passes if actual <  expected
+actual.should be <= expected  # passes if actual <= expected
+actual.should be >  expected  # passes if actual >  expected
+actual.should be >= expected  # passes if actual >= expected
+```
+
+### Other matchers
+```crystal
+actual.should be_close(expected, delta) # passes if actual is within delta of expected:
+                                        # (actual - expected).abs <= delta
+actual.should contain(expected)         # passes if actual.includes?(expected)
+actual.should match(expected)           # passes if actual ==~ expected
```

Comment:
`==~` is not an operator (or I don't know how to use it), I think it should be `=~`.